### PR TITLE
Edits to the Contingent chapter

### PR DESCRIPTION
**Pull request by amyrbrown**
Here are my copyedits on the Contingent chapter.

@brandon-rhodes, there are two very minor queries for you around lines 362 and 1143 on the `Files changed` page. Generally it is a very readable and clear chapter, and was a pleasure to edit!

@MichaelDiBernardo there are a couple of references to figures which should probably be \aosafigrefs, around lines 416, 785, 865, 1383. I flagged a couple of them in `Files changed`.

### CODE REVIEW

File: contingent/contingent.markdown [hunk 1]
```diff
@@ -359,7 +360,7 @@ that can be used without being imported.
 The **tuple** is a read-only sequence
 used to hold heterogeneous data —
 each slot in a tuple typically means something different.
-Here, a tuple (e.g. holds together a hostname and port number,
+Here, a tuple holds together a hostname and port number,
```
Comment (amyrbrown) [inline contingent/contingent.markdown]: It looks to me like the example got lost here -- is there something missing?

File: contingent/contingent.markdown [hunk 2]
```diff
@@ -413,7 +414,7 @@ of the underlying tuples, lists, sets, and dicts.
 Given that each of our graph edges needs
 to know at least its origin node and its destination node,
 the simplest possible representation would be a tuple.
-The top edge in Figure 1 might look like:
```
Comment (amyrbrown) [inline contingent/contingent.markdown]: figref?

File: contingent/contingent.markdown [hunk 3]
```diff
@@ -862,7 +864,7 @@ for the various artifacts in our project's documentation.
 We now have a way for Contingent
 to keep track of tasks and the relationships between them.
 If we look more closely at Figure 2, however,
```
Comment (amyrbrown) [inline contingent/contingent.markdown]: figref?

File: contingent/contingent.markdown [hunk 4]
```diff
@@ -1138,7 +1140,7 @@ Here is what the `task` decorator boilerplate looks like:
 
 This is an entirely typical Python decorator declaration.
 It can then be applied to a function
-by naming it after a `@` character atop the `def`
+by naming it after an `@` character atop the `def`
```
Comment (amyrbrown) [inline contingent/contingent.markdown]: I pronounce `@` "at", so I made this "an", but maybe the Python pronounciation is different and doesn't require "an"?

File: contingent/contingent.markdown (hunk 2 above)
Comment (MichaelDiBernardo) [inline contingent/contingent.markdown]: Oops! I'll fix that.

File: contingent/contingent.markdown (hunk 3 above)
Comment (MichaelDiBernardo) [inline contingent/contingent.markdown]: Also oops.

**Comment by MichaelDiBernardo** [comment]
Also paging @drocco007.

**Comment by drocco007** [comment]
Ooo hey what'd I miss? :D 

I'll have a look at the changes tonight when I get home. Thanks @amyrbrown!

cc: @MichaelDiBernardo

**Comment by brandon-rhodes** [comment]
On a quick read, the edits look good to me! I mean, there are one or two possessive apostrophes that I might not have included, but the style guides disagree with one another, so I'll accept the copyedits. I'll let @drocco007 weigh in if he has any objections.

**Comment by drocco007** [comment]
The edits look good to me as well. Thanks all!

**Comment by MichaelDiBernardo** [comment]
Great, thanks @drocco007 and @brandon-rhodes. Will merge now and follow up with fixes to things I accidentally omitted the first time around.

**Pull request merged**